Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -222,7 +298,33 @@
                 else
                     start = varargin{3};
                 end
-                start = start - SYNCHEPOCH;
+
+                %%% added by Andreas Widmann, widmann@uni-leipzig.de, 2017-05-25
```

Comment:
Drop preambles, postfix. Split overlong comment lines. Check for missing ; in this new block of code.